### ISSUE TITLE
elearning - Supervisor

### ISSUE THREAD

**Issue by carinasoca** [net2phone/uc-roadmap#16]
e-learning tutorial videos to train users on the uContact product and its various roles. 

These videos will be a valuable resource for those interested in learning more about the product and its use, offering an interactive and visual learning experience. 
Additionally, these tutorials will be accessible online, meaning that users can access them anytime and anywhere that is convenient for them. 

These videos will provide a comprehensive understanding on how to effectively utilize the features of uContact to carry out their role.

In summary, the e-learning tutorial videos will be a powerful tool to help users understand and make the most of the uContact features.
